src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the class definition.